app/controllers/course/statistics/aggregate_controller.rb

```diff
@@ -28,33 +30,65 @@ def all_students
   end
 
   def all_assessments
-    @assessments = current_course.assessments.published.includes(tab: :category)
     @all_students = current_course.course_users.students
     @all_submissions_info = all_submissions_info
 
     fetch_all_assessment_related_statistics_hash
   end
 
+  def submission_due
+    @all_students = [CourseUser.where(id: params[:student_id]).first]
```

Comment:
this is because this `submission_due` API only accept one `student_id` from the frontend (the way this API is called is firstly to specify which student we would like to know more).

And the reason I'm using `@all_students` here is because my intention is to reuse the function `all_submissions_info`